librz/bin/format/pyc/marshal.c

```diff
@@ -682,6 +682,10 @@ static pyc_object *get_ascii_object(RzBinPycObj *pyc, RzBuffer *buffer) {
 	ut32 n = 0;
 
 	n = get_ut32(buffer, &error);
+	if (n > ST32_MAX) {
```

Comment:
does the pyc spec accept only positive values here? I mean, is this value interpreted as signed also in the spec?

Reply:
In cpython sources this value is also [compared](https://github.com/python/cpython/blob/main/Python/marshal.c#L1134) with `0x7FFFFFFF`, so I think yes.

Reply:
Great!